addon/components/power-calendar/days.js

```diff
@@ -160,15 +162,18 @@ export default Component.extend({
   buildDay(dayMoment, today, calendar) {
     let id = dayMoment.format('YYYY-MM-DD');
     let momentDate = dayMoment.clone();
+    let isCurrentMonth = momentDate.month() === calendar.center.month();
 
     return {
       id,
       number: momentDate.date(),
       date: momentDate._d,
       moment: momentDate,
+      dow: momentDate.format('ddd').toLowerCase(),
```

Comment:
i've seen "dow" refer to day-of-week a lot but I guess it may not jump out at many people. I'll change that today but one thought is that maybe we should call `dayAbbrev` or `dayCode` as this may help to distinguish it from the more user-oriented uppercased name.